src/SixLabors.Fonts/FontCollectionExtensions.cs

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Six Labors.
+// Licensed under the Apache License, Version 2.0.
+
+namespace SixLabors.Fonts
+{
+    /// <summary>
+    /// Extension methods for <see cref="IFontCollection"/>.
+    /// </summary>
+    public static class FontCollectionExtensions
+    {
+        /// <summary>
+        /// Adds the fonts from the <see cref="SystemFonts"/> collection to this <see cref="FontCollection"/>.
+        /// </summary>
+        /// <param name="collection">The font collection.</param>
+        /// <returns>The <see cref="FontCollection"/> containing the system fonts.</returns>
+        public static FontCollection AddSystemFontCollection(this FontCollection collection)
```

Comment:
```suggestion
        public static FontCollection AddSystemFonts(this FontCollection collection)
```
reads better and prevents users needing to understand that we have a separate collection for system fonts laying around (that feels like implementation detail to me)

Reply:
Done